### DISCUSSION TITLE
Can a single wrapped token on chain A serve multiple foreign chains?

### DISCUSSION THREAD

**Discussion by OliverChuHK** [wormhole-foundation/wormhole#3887] [Developer Q&A] [answered]
For example,  I want to create one wrapped token to be the wrapped token for all USDT tokens on different networks?  Is that possible in wormhole?

Thanks

**Comment by evan-gray** [comment, marked as answer]
You _could_ do something like this with [Native Token Transfers](https://github.com/wormhole-foundation/example-native-token-transfers) by using Locking mode on the chains with existing USDT, but I would caution that there would be potential liquidity issues when leaving the network if the withdrawn amount is greater than the locked amount on a destination chain.

For example, you run the risk that if 1000 were locked on Ethereum and another 1000 on Solana to "YourChain", there would now be 2000 tokens that one could own - and therefore could attempt to transfer out 2000 to Solana which would not be able to be unlocked since there is only 1000 locked there. NTT is entirely flexible though, so you could include checks to mitigate or prevent this in your network of contracts.
